circle.yml

```diff
@@ -1,77 +1,70 @@
 jobs:
-  xcodebuild:
-    macos:
-      xcode: "10.1.0"
-    steps:
-      - checkout
-      - run: git submodule update --recursive --init
-    
-      - run:
-          name: Test UIKit for macOS
-          command: set -o pipefail &&
-            xcodebuild
-              CODE_SIGNING_REQUIRED=NO
-              CODE_SIGN_IDENTITY=""
-              PROVISIONING_PROFILE=""
-              -sdk "macosx"
-              -scheme "UIKit"
-              build test |
-            xcpretty --color --report junit
+    xcodebuild:
+        macos:
+            xcode: "12.5.0"
+        steps:
+            - checkout
+            - run: git submodule update --recursive --init
 
-      - run:
-          name: "send coverage report to codecov"
-          command: bash <(curl -s https://codecov.io/bash) -J 'UIKit'
-          environment:
-            - CODECOV_TOKEN: "4b6cd237-4637-42d6-ad29-0d1a6ee59761"
```

Comment:
hoops, prettier crushed the formatting again. I just updated to a newer xcode version and removed uploading the coverage report because it took 5 minutes and sometimes randomly failed. I think we are not even actively using codecov in this project, no?